config.js

```diff
@@ -2,7 +2,8 @@ const configVars = {
   ADMIN_TOKEN: 'admintoken',
   DATA_TRUST_SERVICE: 'datatrustservice',
   REQUIRE_AUTH: 'requireauth',
-  REQUIRE_AUTH_FOR_OUTGOING: 'requireauthout'
+  REQUIRE_AUTH_FOR_OUTGOING: 'requireauthout',
+  DYANMIC_CLIENT_REGISTRATION: 'dynamicclientregistration'
```

Comment:
typo on the field name, though the value and functions that use it elsewhere are good